commands.yaml

```diff
@@ -42,220 +45,190 @@ azmgmt flow export:
   - '-s, --packageSourceEnvironment [packageSourceEnvironment]'
   - '-f, --format [format]'
   - '-p, --path [path]'
-azmgmt flow get:
+flow get:
   - '-n, --name <name>'
   - '-e, --environment <environment>'
   - '--asAdmin'
-azmgmt flow list:
+flow list:
   - '-e, --environment <environment>'
   - '--asAdmin'
-azmgmt flow run get:
+flow run get:
   - '-n, --name <name>'
   - '-f, --flow <flow>'
   - '-e, --environment <environment>'
-azmgmt flow run list:
+flow run list:
   - '-f, --flow <flow>'
   - '-e, --environment <environment>'
-azmgmt login:
-  - '-t, --authType [authType]'
-  - '-u, --userName [userName]'
-  - '-p, --password [password]'
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-azmgmt connect:
-  - '-t, --authType [authType]'
-  - '-u, --userName [userName]'
-  - '-p, --password [password]'
-  - '-c, --certificateFile [certificateFile]'
-  - '--thumbprint [thumbprint]'
-azmgmt logout:
-azmgmt disconnect:
-azmgmt status:
-graph groupsetting add:
+group setting add:
   - '-i, --templateId <templateId>'
-graph groupsetting get:
+group setting get:
```

Comment:
So group is an interesting one because right now it would be hard to tell what kind of group this is? AAD group, SharePoint group, distribution group? All command names should start with the name of the service they belong to, so in this case, perhaps we should prefix it with `aad`? An alternative could be `o365`, but it could be awkward to use it in non-immersive mode as `o365 o365 group get`. Another idea could be to use `o365group` to disambiguate it with all other groups.

Reply:
yes, that is a pain. In Azure AD they are under just `Groups`. I will let  you decide `aad` or `o365group`

Reply:
Thinking of it, you could argue that the AAD group is one of the pieces of an O365 Group, like email, calendar or planner. Additionally, what we wanted to add support for working with plain AAD groups where you can't have things like a planner or a team? Things could easily get confusing. So perhaps, it's clearer to look something in the are of `o365`: either `o365` as the service, which could be odd since the whole CLI is about O365 or something like `o365group` for the collective thing and `aad group` for AAD groups specifically. What do you think?